### ISSUE TITLE
Dangling reference to fribidi_remove_special_chars

### ISSUE THREAD

**Issue by egmontkob** [fribidi/fribidi#86]
The `fribidi_remove_bidi_marks.3` manual page says: "Use fribidi_remove_special_chars instead."

There's no other occurrance of "fribidi_remove_special_chars" in the source tree.

**Comment by behdad** [comment]
Humm.  Sounds wrong.  I think docs should be updated to remove that and undeprecate `fribidi_remove_bidi_marks`.

**Comment by dov** [comment]
I finally fixed this by undeprecating `fribidi_remove_bidi_marks`. I also undeprecated `fribidi_log2vis()` as I thought it absurd that `README.md` refered to a deprecated function.

**Comment by egmontkob** [comment]
`fribidi_log2vis()`:

> This function is obsolete because it only handles one-line paragraphs.

I found it absolutely reasonable to deprecate such a function for this reason. Anyway, you did not remove the words "obsolete", "deprecated" from its API doc. If you really wish to undeprecate it, you might want to rephrase it to something like a "warning" or "note".

**Comment by dov** [comment]
Thanks. I updated the comment to read:
```
 * Note that this function handles one-line paragraphs. For multi-
 * paragraph texts it is necessary to first split the text into
 * separate paragraphs and then carry over the resolved pbase_dir
 * between the subsequent invocations.
```

**Comment by egmontkob** [comment]
Thanks a lot!
